axum-extra/src/extract/query.rs

```diff
@@ -51,6 +51,35 @@ use std::fmt;
 /// example.
 ///
 /// [example]: https://github.com/tokio-rs/axum/blob/main/examples/query-params-with-empty-strings/src/main.rs
+///
+/// While `Option<T>` will handle empty parameters (e.g. `param=`), beware when using this with a
+/// `Vec<T>`. If your list is optional, use `Vec<T>` in combination with `#[serde(default)]`
+/// instead of `Option<Vec<T>>`. `Option<Vec<T>>` will handle 0, 2, or more arguments, but not one
+/// argument. For details, see [the documentation of `serde_html_form`].
```

Comment:
What extra details are you talking about here? 😅

Reply:
The crate level documentation, which originally mentioned this detail. If you find it unnecessary though I can remove it again :slightly_smiling_face:

Reply:
Yeah it does mention this, but doesn't go into more detail than what you wrote here. I think it's better to remove it to avoid confusion.